cmd/rollkit/commands/toml.go

```
@@ -25,11 +26,11 @@ func NewTomlCmd() *cobra.Command {
 
 var initCmd = &cobra.Command{
 	Use:   "init",
-	Short: fmt.Sprintf("Initialize a new %s file", rollconf.RollkitToml),
-	Long:  fmt.Sprintf("This command initializes a new %s file in the current directory.", rollconf.RollkitToml),
+	Short: fmt.Sprintf("Initialize a new %s file", rollconf.RollkitConfigToml),
+	Long:  fmt.Sprintf("This command initializes a new %s file in the current directory.", rollconf.RollkitConfigToml),
 	Run: func(cmd *cobra.Command, args []string) {
```

Nit, use `RunE` and return errors instead of printing + exit 1.